.github/workflows/build.yml

```diff
@@ -476,3 +524,38 @@ jobs:
 
       - name: Run jest tests
         run: yarn jest
+
+  collate_simplecov_results:
+    runs-on: ubuntu-22.04
+    needs:
+      - controllers
+      - models
+      - engines
+      - system_admin
+      - system_consumer
+      - test_the_rest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true # runs 'bundle install' and caches installed gems automatically
```

Comment:
I guess we only need to use the simplecov gem, and don't need all the other gems in the gemfile.
In that case, perhaps this could be optimised. But hopefully loading all the gems from cache doesn't take too long anyway.